### PR TITLE
adding code coverage

### PR DESCRIPTION
**Pull request by KjellKod**
screen shot 2016-01-12 at 10 29 51 am

AlarmClock, which is all template does not get counted in the code coverage. I am not sure why :(

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -22,61 +22,43 @@ SET(ACTIVE_CPP0xx_DIR "Release")
...
-      MESSAGE(STATUS "- MSVC: Set variadic max to 10 for MSVC compatibility")
-      # Remember to set set target properties if using GTEST similar to done below on target "unit_test"
-      # "set_target_properties(unit_test PROPERTIES COMPILE_DEFINITIONS "GTEST_USE_OWN_TR1_TUPLE=0")
-   MESSAGE("")
-   MESSAGE("Windows: Please run the command [cmake -DCMAKE_BUILD_TYPE=Release -G \"Visual Studio 11\" ..]")
-   MESSAGE("if cmake finishes OK, do 'msbuild TestOfStopWatch.sln /p:Configuration=Release'")
-   MESSAGE("then run 'Release\\TestOfStopWatch.exe'")
-   MESSAGE("")
+   SET(PLATFORM_LINK_LIBRIES rt)
+   SET(CMAKE_CXX_FLAGS "-Wall -rdynamic -Wunused -std=c++11 -pthread -D_GLIBCXX_USE_NANOSLEEP")
```
Comment (john-gress) [inline CMakeLists.txt]: I think you can drop ```-pthread```

**Comment by john-gress** [comment]
:+1:

File: CMakeLists.txt (hunk 1 above)
Comment (KjellKod) [inline CMakeLists.txt]: Maybe. It used to be required if you run threaded code.

Comment (KjellKod) [inline CMakeLists.txt]: leaving it in for now due to the std:;thread issues

**Pull request merged**